### PR TITLE
update: cosign from v0.6.0 to v1.0

### PR DESCRIPTION
**Pull request by xopham**

### CODE REVIEW

File: docs/validators/sigstore_cosign.md [hunk 1]
```diff
@@ -7,13 +7,9 @@ You can read more about it [here](https://blog.sigstore.dev/).
 Connaisseur currently supports the elementary function of verifying Cosign-generated signatures against the locally created corresponding public keys.
 We plan to expose further features of Cosign and Sigstore in upcoming releases, so stay tuned!
 
-> :warning: Sigstore and Cosign are currently in *pre-release* state and under heavy development and so is our support for them. We therefore consider this an *experimental feature* that might unstable over time. As such, it is not part of our semantic versioning guarantees and we take the liberty to adjust or remove it with any version at any time without incrementing MAJOR or MINOR.
```
Comment (annekebr) [inline docs/validators/sigstore_cosign.md:10]: Is our support for cosign not experimental anymore?

File: docs/validators/sigstore_cosign.md [hunk 2]
```diff
@@ -7,13 +7,9 @@ You can read more about it [here](https://blog.sigstore.dev/).
...
-
 ## Basic usage
 
 Getting started with Cosign is very well described in the [docs](https://github.com/sigstore/cosign).
-Please check there for detailed instructions.
-The currently used version can be found in our [Makefile](https://github.com/sse-secure-systems/connaisseur/blob/master/Makefile/#L5).
```
Comment (annekebr) [inline docs/validators/sigstore_cosign.md:16]: Why is this removed?

**Review by annekebr — COMMENTED** [review]
as the `update:` prefix is newly introduced you should add it in the Contributing guide

File: docs/validators/sigstore_cosign.md (hunk 1 above)
Comment (xopham) [reply docs/validators/sigstore_cosign.md:10]: imho I feel good about it and consider it tested and secure. Anything that strikes you as experimental?

**Review by phbelitz — APPROVED** [review]
:smirk_cat:
